client/src/components/AvatarDisplay/useStyles.ts

```diff
@@ -19,6 +19,16 @@ const useStyles = makeStyles((theme: Theme) =>
     large: {
       width: theme.spacing(7),
       height: theme.spacing(7),
+      '&:hover': {
+        cursor: 'pointer',
+      },
```

Comment:
You can just make the css has `cursor: 'pointer'` instead of setting that on the hover. It'll always be clickable.

Reply:
Got it!